.github/workflows/build-and-test.yml

```diff
@@ -12,7 +12,6 @@ jobs:
     name: Build and Test
     uses: Lombiq/GitHub-Actions/.github/workflows/build-and-test-orchard-core.yml@dev
     with:
-      machine-types: "['gitrunners-ubuntu-2204-x64-4vcpu']"
       timeout-minutes: 20
       build-create-binary-log: "true"
       blame-hang-timeout: "5m"
```

Comment:
Do we really need these extra parameters here?

Reply:
We don't actually need these I guess.